hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -619,6 +631,9 @@ private Response createMultipartKey(String bucket, String key, long length,
       if (ex.getResult() == ResultCodes.NO_SUCH_MULTIPART_UPLOAD_ERROR) {
         throw S3ErrorTable.newError(NO_SUCH_UPLOAD,
             uploadID);
+      } else if (ex.getResult() == ResultCodes.PERMISSION_DENIED) {
```

Comment:
Is it possible to have a common helper that does the OM result code => s3 error translation?

Reply:
In long term，it's good to have such a mapping table.  
But there are many details need further discussion, for example, a error response has HTTP error code and s3 error code.  Multiple s3 error codse will map to the same HTTP error code.
Some OM result codes, like QUOTA_EXCEEDED, I quickly go through all HTTP codes, didn't find a perfect match code. 
Other OM result codes, such as NOT_SUPPORTED_OPERATION, is just translated to "interval error" in s3g.